src/Components/Blazor/WebAssembly.Authentication/src/Services/RemoteAuthenticationService.cs

```diff
@@ -119,8 +125,14 @@ public virtual async ValueTask<AccessTokenResult> GetAccessToken(AccessTokenRequ
         }
 
         /// <inheritdoc />
-        public virtual async Task<ClaimsPrincipal> GetCurrentUser()
+        protected virtual async ValueTask<ClaimsPrincipal> GetCurrentUser()
```

Comment:
I like the 1-minute caching you've put in here. That feels like a good balance of efficiency and freshness.

Just so I can explain this to other people, can you summarise what it is that happens each time the 1-minute cache expires? What will we say in docs? Something like: *The next time we evaluate authentication status, the system will make an HTTP request to the remote authentication service to verify that the user is still authenticated, and to update the list of claims associated with their `ClaimsPrincipal`*. Is that accurate?